lib/matplotlib/patches.py

```diff
@@ -4415,25 +4412,22 @@ def draw(self, renderer):
         if not self.get_visible():
             return
 
-        with self._bind_draw_path_function(renderer) as draw_path:
-
-            # FIXME : dpi_cor is for the dpi-dependency of the linewidth. There
-            # could be room for improvement.  Maybe _get_path_in_displaycoord
-            # could take a renderer argument, but get_path should be adapted
-            # too.
-            self._dpi_cor = renderer.points_to_pixels(1.)
-            path, fillable = self._get_path_in_displaycoord()
+        # FIXME: dpi_cor is for the dpi-dependency of the linewidth.  There
+        # could be room for improvement.  Maybe _get_path_in_displaycoord could
+        # take a renderer argument, but get_path should be adapted too.
+        self._dpi_cor = renderer.points_to_pixels(1.)
```

Comment:
So its OK now for `self._dpi_cor` to be changed permanently here?

Reply:
It was already set permanently before?